packages/npm-package-json-lint-config/CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-## v1.1.3 (2018-09-05)
+## 1.2.0 (2019-03-01)
+
+### New Feature
+
+- The `npm-package-json-lint` peer dependency has been updated from requiring `^3.3.1` to requiring `^3.6.0`, adding [a rule which enforces the presence of the `repository.directory` field](https://github.com/tclindner/npm-package-json-lint/pull/106) [#14200](https://github.com/WordPress/gutenberg/pull/14200)
```

Comment:
Would you consider the new enforcement to be a breaking change? I might.

We do similar with the default ESLint config. It seems a problem for any "config" modules, where most _any_ change to the configuration could be considered breaking.

Reply:
That's a good question. I had been thinking along the lines that since this PR follows #14059 it wouldn't break anything, but technically it is a breaking change for future work as it adds a new requirement that wasn't there before. I'd be happy to revise accordingly if you think that makes more sense.

Reply:
It is **not** a breaking change because of my comment below, the `repository.directory` rule should **not** be shipped in the package as most consumers of the `@wordpress/npm-package-json-lint-config` package do not use Lerna, it should be added to the _root_ `package.json` `npm-package-json-lint` configuration in https://github.com/WordPress/gutenberg/blob/master/package.json#L124-L150 and **not** distriubuted via the _package_ configuration.

Further, because the _distributed_ configuration is **not** changing, only a _patch_ release is required, so `1.1.4` instead of `1.2.0`

Reply:
Thanks @aduth and @ntwb. I've applied @ntwb's suggestions so this is no longer a breaking change.